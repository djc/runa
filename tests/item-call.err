item-call.rns [2.9]: object is not a function
    args[0]()
        ^
