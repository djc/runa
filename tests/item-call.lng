def main(name: str, args: array[str]) -> int:
	args[0]()
