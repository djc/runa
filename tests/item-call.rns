def main(name: &Str, args: &Array[Str]):
	args[0]()
