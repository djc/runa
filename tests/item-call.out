tests/item-call.lng [2.9]: not a function or method
    args[0]()
           ^
